.github/actions/update-check/action.yml

```diff
@@ -83,21 +83,31 @@ runs:
             exclude_pull_requests: true
           });
 
-          const message = "The CI workflow run with tests that require additional secrets finished as " + process.env.conclusion + ": " + run.html_url
+          const started = "The CI workflow run with tests that require additional secrets has been started: " + run.html_url
+          const finished = "The CI workflow run with tests that require additional secrets finished as " + process.env.conclusion + ": " + run.html_url
           const comments = await github.paginate(github.rest.issues.listComments.endpoint.merge({
             ...context.repo,
             issue_number: process.env.number
           }))
-          const exists = comments.filter(comment => comment.body === message).length != 0
+          const comment = comments.find(comment => comment.body === started || comment.body === finished)
+
+          if (comment !== undefined) {
+            if (comment.body === finished) {
+              return;
+            }
```

Comment:
Looks unreachable. Even if you comment twice for same SHA two different runs will be started if the automation hasn't posted a comment yet

Reply:
This action is called from the test jobs created from a matrix, so there can be more than one. This is done to signal the original PR about the failure as early as possible.

Reply:
this one deserves comment IMO.

Reply:
In the workflow?

Reply:
yes, in the body of this `if` block would make sense